website/src/content/docs/docs/libraries/http/multipart.md

```diff
@@ -33,42 +44,136 @@ Content-Type: application/octet-stream
 
 By default, the `Content-Type` of individual request parts is set automatically according to the type of the schema properties that describe the request parts:
 
-| Schema Property Type                     | Content-Type               | Example                       |
-| ---------------------------------------- | -------------------------- | ----------------------------- |
-| Primitive                                | `text/plain`               | `string`, `number`, `boolean` |
-| Complex value or array of complex values | `application/json`         | `Address`, `Address[]`        |
-| `bytes`                                  | `application/octet-stream` | `bytes`                       |
+| Schema Property Type                     | Content-Type               | Example                                                          |
+| ---------------------------------------- | -------------------------- | ---------------------------------------------------------------- |
+| Primitive                                | `text/plain`               | `HttpPart<string>`<br> `HttpPart<number>`<br>`HttpPart<boolean>` |
+| Complex value or array of complex values | `application/json`         | `HttpPart<Address>`<br>`HttpPart<Address[]>`                     |
+| `File`, `bytes`                          | `application/octet-stream` | `HttpPart<bytes>`<br>`HttpPart<bytes>`                           |
```

Comment:
Not sure why `HttpPart<bytes>` is written twice?

Reply:
meant to have one as File